WebServers/HTTPServer/HttpProcessor.cs

```diff
@@ -88,7 +88,7 @@ public void HandleClient(TcpClient tcpClient, int ListenerPort)
                     {
                         if (tcpClient.Available > 0 && outputStream.CanWrite)
                         {
-                            HttpRequest? request = GetRequest(inputStream, clientip, clientport.ToString());
+                            HttpRequest request = GetRequest(inputStream, clientip, clientport.ToString());
```

Comment:
Please add this back, the request must be nullable for the plugin system

Reply:
Ahh ok

Reply:
Any particular reason why a request should be nullable for the plugin system? a HTTPRequest will NEVER be null.

Reply:
Yes I got confused with the response variable, you are right, request cannot be null anymore, it could have been before.

Reply:
The warning situation is a bump from net core 3.1 to net6.0 that changed the nullables attributes. Horizon and some nuget are responsible for this yeah.